packages/color-scale-block/src/components/EmptyView.tsx

```diff
@@ -0,0 +1,38 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+type EmptyViewProps = {
+    placeholderBackgroundColor: string;
+    height: string;
+    totalNumberOfBlocks: number;
+    index: number;
+    isFirst: boolean;
+    isLast: boolean;
+    width: number;
+};
+
+export const EmptyView = ({
+    placeholderBackgroundColor,
+    totalNumberOfBlocks,
+    index,
+    isFirst,
+    isLast,
+    height,
+    width,
+}: EmptyViewProps) => {
+    return (
+        <div
+            style={{
+                backgroundColor: placeholderBackgroundColor,
+                width: `${width}px`,
+                height,
+            }}
+            key={`placeholder-${index}`}
+            className={`empty-view tw-flex tw-overflow-hidden
+            ${isFirst ? 'tw-pl-[1px]' : ''} ${index === totalNumberOfBlocks - 1 ? 'tw-pr-[1px]' : ''}
```

Comment:
This seems unnecessary. You can probably use `last:` from tailwind and remove the props.